xcube/core/store/storepool.py

```diff
@@ -295,6 +315,40 @@ def get_store_instance(self, store_instance_id: str) -> DataStoreInstance:
         self._assert_valid_instance_id(store_instance_id)
         return self._instances[store_instance_id]
 
+    # def get_store(self,
+    #               store_instance_id: str = None,
+    #               store_config: DataStoreConfig = None
+    #               ) -> DataStore:
+    #     if store_instance_id is not None:
+    #         if store_config is not None:
+    #             raise ValueError('Not both parameters "store_instance_id "'
+    #                              'and "store_config" can be given')
+    #         self._assert_valid_instance_id(store_instance_id)
+    #         return self._instances[store_instance_id].store
+    #     elif store_config is not None:
+    #         store_instance_id = self.get_store_instance_id(store_config)
+    #         return self._instances[store_instance_id].store
+    #     else:
+    #         raise ValueError('Either parameter "store_instance_id" or '
+    #                          '"store_config" must be given.')
+    #
+    # def get_store_instance(self,
+    #                        store_instance_id: str = None,
+    #                        store_config: DataStoreConfig = None
+    #                        ) -> DataStoreInstance:
+    #     if store_instance_id is not None:
+    #         if store_config is not None:
+    #             raise ValueError('Not both parameters "store_instance_id "'
+    #                              'and "store_config" can be given')
+    #         self._assert_valid_instance_id(store_instance_id)
+    #         return self._instances[store_instance_id]
+    #     elif store_config is not None:
+    #         store_instance_id = self.get_store_instance_id(store_config)
+    #         return self._instances[store_instance_id]
+    #     else:
+    #         raise ValueError('Either parameter "store_instance_id" or '
+    #                          '"store_config" must be given.')
+
```

Comment:
Why has this been added/removed and/or commented out?

Reply:
This was an early implementation to determine stores based on a storeconfig. I found that the newly added method get_store_instance_id was enough, so I commented these out but forgot to remove them. Removing them now.